code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -176,6 +176,8 @@
 
 	for(var/t in adjacent_turfs)
 		var/turf/open/enemy_tile = t
+		if(!enemy_tile || istype(enemy_tile))
```

Comment:
Yeah, no this is not a proper fix, this adds cost to every process, proper solution is locking atmos changes on turfs that are currently loaded.

Reply:
We already "is locking atmos changes on turfs that are currently loaded" 
Problem in the outer border turfs. It try to interract with the turfs that we load before it propertly loaded, that creates runtimes.

Or i dont complite understand actual turf atmos updating mechanick.
As i think, now turf chechs the list adjacent_turfs, what changes when changed adjacent turf initializes. But template first place all map and next init it, that creates runtimes.

I see 2 ways to fix this.
First. Create a outer border of the inited not interracting turfs.

Second. Change the loader to disable or freeze outer bouder turfs.

Reply:
Add special flag to ChangeTurf that freeze adjacent turfs or remove turf from adjacent_turfs?